docs/src/main/asciidoc/chapters/administration.txt

```diff
@@ -1182,17 +1182,19 @@ conditions that vary from the environments individually tested in the various
 components. For example, Accumulo's use of HDFS includes many short block
 reads, which differs from the more common full file read used in most
 map/reduce applications. We have found that certain versions of Accumulo and
-Hadoop will include stability bugs that greatly affect overall stability. In
-our testing, Accumulo 1.6.2, Hadoop 2.6.0, and Zookeeper 3.4.6 resulted in a
-stable VM clusters that did not fail a month of testing, while Accumulo 1.6.1,
-Hadoop 2.5.1, and Zookeeper 3.4.5 had a mean time between failure of less than
-a week under heavy ingest and query load. We expect that results will vary with
-other configurations, and you should choose your software versions with that in
-mind.
-
-
-
-
-
-
-
+Hadoop will include stability bugs that greatly affect overall stability.
+Release notes typically contain information about versions used in
+release testing.
+
+The following table shows the Hadoop, Zookeeper and
+Thrift versions defined in the dependency section of the POM for building the
+artifacts.  
+
+|================================================
+|Accumulo |Hadoop |Zookeeper |Thrift
+|1.7            |2.2.0     |3.4.6          |0.9.1
+|1.8            |2.6.4     |3.4.6          |0.9.3
+|================================================
+
```

Comment:
> Wasn't suggesting that.

Did not think you were. Just explaining myself, and roughly outlining where I think we should draw the line.

I'd personally only feel obligated to update the latest in each (1.7.4, 1.8.1, 1.9.0 in your example). But, I'd feel more comfortable about that if we had a prominent "This version has been obsoleted by version x.y.z" at the top of each obsoleted release notes page.